.gitignore

```diff
@@ -29,3 +29,6 @@ nohup.out
 
 # Hugo output
 public/
+
+# User-specific editorconfig files
+.editorconfig
```

Comment:
What kind of stuff would go in here? You can set your editor in your ~/.gitconfig or <repo>/git/config, which don't need to go in the repo's base directory and thus don't need to be ignored.

Reply:
Check the URL in the description for more info. Editor configs are a standardized format for configuring a wide variety of editors (VS Code, Sublime, etc).